guides/v2.2/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -86,19 +86,25 @@ Any change not listed below is considered a PATCH level change.
 | | Changed format of the returned method result | MAJOR|
 | | New method exception (excluding subtypes of existing exceptions) | MAJOR|
 | | New method exception (subtypes of an existing one) | PATCH|
-| **JavaScript Interface** (marked with `@api`) | New interface| MINOR|
+| **JavaScript class** (marked with `@api`)| New class| MINOR|
 | | New method added | MINOR|
-| | Interface removed| MAJOR|
+| | New argument added | MINOR|
+| | Class removed| MAJOR|
 | | Method removed | MAJOR|
-| | New required method argument | MAJOR|
-| | Changed method signature | MAJOR|
-| | Last argument added| MINOR|
-| **JavaScript class** (marked with `@api`| New class| MINOR|
+| | Argument removed | MAJOR|
+| | New event| MINOR|
+| | Renamed event| MAJOR|
+| | Removed event| MAJOR|
+| | New event property | MINOR|
+| | Changed event property | MAJOR|
+| | Removed event property | MAJOR|
+| | Changed event ordering | MAJOR|
+| **JavaScript modules** (marked with `@api`)| New module| MINOR|
```

Comment:
Do you think it's worth putting `JavaScript Modules (AMD Modules)` to be clear?

I say that because, when I go "magento javascript modules," my first result is https://devdocs.magento.com/videos/fundamentals/add-a-javascript-module/, which is about creating a _Magento_ module. Just worry if we're overloading that term

Reply:
Good call, I will go ahead and make that distinction.